packages/davinci-client/package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "@forgerock/davinci-client",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "main": "./dist/index.js",
   "module": "./dist/index.js",
   "typings": "./dist/index.d.ts",
   "type": "module",
   "files": ["dist"],
+  "publishConfig": {
```

Comment:
This is an `npm` requirement for publishing with provenance.

We either set it here, or add an ENV variable in the pipeline to allow this

https://github.com/nrwl/nx/issues/27754#issuecomment-2327574037

```
NPM_CONFIG_ACCESS: public

```